astrokat/simulate.py

```diff
@@ -287,27 +288,43 @@ def scan(
     def slew_time(self, target):
         """Get slew time.
 
-        How long in seconds, it took to the antennas to move
-        from previous target to the next
+        How long in seconds, it took for the antennas to move
+        from current target to the next.
+
+        Notes:
+            The slew time is calculated with consideration of az-el motion
+            of antennas instead of the current angular distance between sources,
+            assuming a slew_speed 2 deg/s. Antennas slew at 2 deg/s in az while
+            moving at 1 deg/s in el. There is about 3 seconds for overhead
 
         Parameters
         ----------
          target: katpoint.Target
              a description which contains parameters such as the
              target name, position, flux model.
 
+        Returns
+        -------
+        slew_time: float
+            The number of seconds it takes to slew
+
         """
-        slew_speed = 2.0  # degrees / sec
-        self.katpt_current.body.compute(self.katpt_current.antenna.observer)
-        target.body.compute(target.antenna.observer)
-        try:
-            separation_angle = ephem.separation(self.katpt_current.body, target.body)
-        # TODO: need to find a clean implementation with
-        # ephem_extra.StationaryBody
-        except TypeError:
-            slew_time = _DEFAULT_SLEW_TIME
-        else:
-            slew_time = numpy.degrees(separation_angle) / slew_speed
+        now = timestamp2datetime(self.time)
+        az1, el1 = self.katpt_current.azel(ephem.Date(now))
+        az2, el2 = target.azel(ephem.Date(now))
+
+        az1 = katpoint.rad2deg(az1)
+        el1 = katpoint.rad2deg(el1)
+        az2 = katpoint.rad2deg(az2)
+        el2 = katpoint.rad2deg(el2)
+
+        az_dist = numpy.abs(az2 - az1)
+        el_dist = numpy.abs(el2 - el1)
+
+        az_dist = az_dist if az_dist < 180. else 360. - az_dist
+
+        slew_time = max(0.5 * az_dist, 1.0 * el_dist) + 3
+
         return slew_time
```

Comment:
Earlier today there was some discussion about including the target azimuth and elevation in the user log lines, so it was being returned here.  What happened to that idea?